### PR TITLE
Missing actions for call

### PR DESCRIPTION
**Pull request by avbel**

### CODE REVIEW

File: lib/call.js [hunk 1]
```diff
@@ -369,6 +413,120 @@ var Call = function (client) {
 		})
 		.asCallback(callback);
 	};
+
+	/**
+	 * Get events for the call.
+	 * @param {String} callId The ID of the call
+	 * @param {Function} [callback] Callback with the event list
+	 * @return {Array.<EventResponse>} A promise for the event list
```
Comment (dtolb) [inline lib/call.js:421]: Missing `eventResponse` in docs

File: lib/call.js [hunk 2]
```diff
@@ -369,6 +413,120 @@ var Call = function (client) {
...
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Get a single event for the call.
+	 * @param {String} callId The ID of the call
+	 * @param {String} eventId The ID of the event to get
+	 * @param {Function} [callback] Callback with the event list
+	 * @return {EventResponse} A promise for the event list
```
Comment (dtolb) [inline lib/call.js:444]: Missing `eventResponse` in docs

File: lib/call.js [hunk 3]
```diff
@@ -369,6 +413,120 @@ var Call = function (client) {
...
+			return response.body;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Get recordings for the call.
+	 * @param {String} callId The ID of the call
+	 * @param {Function} [callback] Callback with the recording list
+	 * @return {Array.<RecordingResponse>} A promise for the recording list
```
Comment (dtolb) [inline lib/call.js:466]: Missing `recordingResponse` in docs... not sure if it makes sense to link to [Recording Response](https://github.com/bandwidthcom/node-bandwidth/blob/2.0-preview/lib/recording.js#L119) in recording. Are the payloads the same?

File: lib/call.js [hunk 4]
```diff
@@ -369,6 +413,120 @@ var Call = function (client) {
...
+	 * @param {String} callId The ID of the call
+	 * @param {Function} [callback] Callback with the transcription list
+	 * @return {Array.<RecordingResponse>} A promise for the transcription list
+	 * @example
+	 * // Promise
+	 * client.Call.getTranscriptions(callId).then(function (list) {});
+	 * // Callback
+	 * client.Call.getTranscriptions(callId, function (err, list) {});
+	 */
+	this.getTranscriptions = function (callId, callback) {
```
Comment (dtolb) [inline lib/call.js:495]: Missing `transcriptionResponse` in docs... not sure if it makes sense to link to [transcriptons Response](https://github.com/bandwidthcom/node-bandwidth/blob/2.0-preview/lib/recording.js#L132) in recording. Are the payloads the same?

File: lib/call.js [hunk 5]
```diff
@@ -369,6 +413,120 @@ var Call = function (client) {
...
+			return response.body;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Get transcriptions for the call.
+	 * @param {String} callId The ID of the call
+	 * @param {Function} [callback] Callback with the transcription list
+	 * @return {Array.<RecordingResponse>} A promise for the transcription list
```
Comment (dtolb) [inline lib/call.js]: Recording response or transcritption response?

Comment (avbel) [inline lib/call.js]: Fixed

**Comment by avbel** [comment]
Done

**Pull request merged**